extensions/blocks/membership-button/edit.jsx

```diff
@@ -154,6 +154,20 @@ class MembershipsButtonEdit extends Component {
 		);
 	};
 
+	renderAmount = product => {
+		const amount = formatCurrency( parseFloat( product.price ), product.currency );
+		if ( product.interval === '1 month' ) {
+			return sprintf( __( '%s /month', 'jetpack' ), amount );
+		}
+		if ( product.interval === '1 year' ) {
+			return sprintf( __( '%s /year', 'jetpack' ), amount );
+		}
+		if ( product.interval === '1 year' ) {
```

Comment:
Was this additional check left behind? Could we remove it?

Reply:
This was reserved for one-time subscriptions. System has capability to handle those and we can throw them in the mix later.

Reply:
I am just bad at copypasting - I fixed the comparison here. Nice catch!